browser/pages/confirm/confirm.html

```diff
@@ -374,11 +374,16 @@
       <span>Red Hat Container Development Kit requires Oracle VirtualBox {{checkboxModel.virtualbox.minimumVersion}} or newer 5.1.x release. Please uninstall detected Oracle VirtualBox and restart Installer to continue.</span>
     </div>
 
-    <div class="message-container error-message" ng-show="!confCtrl.isDisabled && !checkboxModel.hyperv.isConfigured() && checkboxModel.virtualbox.virtualizationEnabled === false && (checkboxModel.cdk.selectedOption == 'install' || checkboxModel.virtualbox.selectedOption == 'install')">
+    <div class="message-container error-message" ng-show="!confCtrl.isDisabled && !checkboxModel.hyperv.isConfigured() && checkboxModel.virtualbox.virtualizationEnabled === false && (checkboxModel.cdk.selectedOption == 'install' || checkboxModel.virtualbox.selectedOption == 'install') && platform === 'win32'">
```

Comment:
Check for platform should not be required, because we still following the same rules:
1. Promise resolved to false - error;
2. Promise resolved to true - everything is fine;
3. Promise resolved to undefined - warning.

Even though we going to have warning on macOS when virtualization is supported, it is fine, because we cannot say it is turned on or not.